spec.json

```diff
@@ -10,7 +10,7 @@
       "basic_auth_creds": {
         "title": "Credentials (BasicAuth)",
         "type": "ObjectType",
-        "description": "When making a request that needs Basic Authentication, enter the username and password.",
+        "description": "When making a request that needs basic authentication, enter the username and password.",
```

Comment:
I actually think that Basic Authentication is correct here. Google "basic auth vs oauth" and you will see it being referred to as a title.